gcp/api/server.py

```diff
@@ -435,6 +435,13 @@ def determine_version(version_query: osv_service_v1_pb2.VersionQuery,
                                         len(version_query.file_hashes))
 
 
+@ndb.tasklet
+def valid_ecosystems():
```

Comment:
Should we delete this now?

Reply:
Yep! Removed in https://github.com/google/osv.dev/pull/1485/commits/10e894be6f4d9b506f5459ea296592c312f5c053